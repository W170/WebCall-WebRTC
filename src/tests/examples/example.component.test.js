// ComponentName.test.js
import React from 'react'
import { shallow } from 'enzyme'
import ExampleComponent from '../../components/examples/example.component'

describe('ExampleComponent', () => {
  it('should render my component', () => {
    const wrapper = shallow(<ExampleComponent />)

    expect(wrapper.getElements()).toMatchSnapshot()
  })
})
