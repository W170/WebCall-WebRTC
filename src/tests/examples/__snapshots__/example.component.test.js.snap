// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExampleComponent should render my component 1`] = `
Array [
  <Connect(ExampleComponent)
    i18n={Object {}}
    t={[Function]}
    tReady={false}
  />,
]
`;
